src/features/PublicityToggler/index.tsx

```diff
@@ -9,9 +9,10 @@ import {useEnhance} from './hooks';
 type Props = {
   id: WidgetId;
   controllable?: boolean;
+  labelClassName?: string;
```

Comment:
mb go for component injection instead of classname injection?

Reply:
looks like overengineering here